### DISCUSSION TITLE
If you can now support shizuku , would ADB WiFI access buy us anything?

### DISCUSSION THREAD

**Discussion by telrod11** [d4rken-org/sdmaid-se#653] [General]
I don't use shizuku , but do have Tasker providing ADB Wifi through LADB.

Would this enhance SDMaid SE in any way?

(Just bought your app again, and I do believe it worth paying for again)

:)

**Comment by d4rken** [comment]
In itself no, but if you have ADB you can easily set up Shizuku which would allow SD Maid to fully utilize ADB.

The ADB shell user itself has more permissions than a normal app/user, Shizuku does not grant more or less permissions it's just a tool that helps other apps launch code with the same permission as the ADB user.

**Reply by telrod11** [reply]
Ah, thanks for the clarification
